recipe/meta.yaml

```diff
@@ -7,20 +8,21 @@ package:
 source:
   fn: traitlets-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/t/traitlets/traitlets-{{ version }}.tar.gz
-  md5: d0725f9818bd9e112a6531da26d433b8
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install
```

Comment:
That another issue. While PyPA is moving towards making `pip` and `setuptool` one with the basic python distribution conda-forge went in the opposite direction by removing them.

So you need to add `pip` as a build dependency.